src/web/views/FlowBuilder/sidePanel/properties/ActionModalForm.jsx

```diff
@@ -132,9 +132,13 @@ class ActionModalForm extends Component {
           <Select
             name="functionToInvoke"
             value={this.state.functionInputValue}
-            options={functionSuggestions}
+            options={availableFunctions.map(x => ({ label: x.name, value: x.name }))}
             onChange={val => {
-              this.setState({ functionInputValue: val && val.value })
+              const { metadata } = availableFunctions.find(fn => fn.name === (val && val.value)) || {}
```

Comment:
In the case where arguments where already filled in, could we show a "confirm()" that asks the user if he wants the params to be overwritten?

Reply:
@slvnperron , done!